test/semantics/modfile12.f90

```diff
@@ -48,8 +48,8 @@ subroutine bar(x)
 !  integer(4),parameter::n=3_4
 !  integer(4),parameter::l=-3_4
 !  real(4),parameter::e=3.333333432674407958984375e-1_4
-!  real(4)::x(1_4:13_8)
-!  real(4)::y(1_8:8_4)
+!  real(4)::x(int(1_4,kind=8):13_8)
```

Comment:
Can `int(1_4,kind=8)` be folded to `1_8`?

Reply:
Will do.

Reply:
Done.